staging/src/k8s.io/api/apps/v1beta2/types.go

```diff
@@ -681,7 +681,7 @@ type RollingUpdateDaemonSet struct {
 	// pod is available (Ready for at least minReadySeconds) the old DaemonSet pod
 	// on that node is marked deleted. If the old pod becomes unavailable for any
 	// reason (Ready transitions to false, is evicted, or is drained) an updated
-	// pod is immediatedly created on that node without considering surge limits.
+	// pod is immediately created on that node without considering surge limits.
```

Comment:
There will be generated file. https://github.com/search?q=repo%3Akubernetes%2Fkubernetes%20immediatedly&type=code 

You may fix them all in this PR.

Reply:
Done :)